arch/risc-v/src/esp32c3/Kconfig

```diff
@@ -283,4 +297,83 @@ endif # ESP32C3_UART1
 
 endmenu
 
+menu "Real-Time Timer"
+	depends on ESP32C3_RT_TIMER
+
+config ESP32C3_RT_TIMER_TASK_NAME
+	string "Timer task name"
+	default "rt_timer"
+
+config ESP32C3_RT_TIMER_TASK_PRIORITY
+	int "Timer task priority"
+	default 223 # Lower than high priority workqueue
+
+config ESP32C3_RT_TIMER_TASK_STACK_SIZE
+	int "Timer task stack size"
+	default 2048
+
+endmenu # Real-Time Timer
+
+menu "WiFi configuration"
+	depends on ESP32C3_WIRELESS
+
+config ESP32C3_WIFI_STATIC_RXBUF_NUM
+    int "WiFi static RX buffer number"
+	default 10
+
+config ESP32C3_WIFI_DYNAMIC_RXBUF_NUM
+    int "WiFi dynamic RX buffer number"
+	default 32
+
+config ESP32C3_WIFI_DYNAMIC_TXBUF_NUM
+    int "WiFi dynamic TX buffer number"
+	default 32
+
+config ESP32C3_WIFI_TX_AMPDU
+	bool "WiFi TX AMPDU"
+	default y
+
+config ESP32C3_WIFI_RX_AMPDU
+	bool "WiFi RX AMPDU"
+	default y
+
+config ESP32C3_WIFI_RXBA_AMPDU_WZ
+    int "WiFi RX BA AMPDU windown size"
+	default 6
+
+config ESP32C3_WLAN_RXBUF_NUM
+	int "WLAN netcard RX buffer number"
+	default 16
+
+config ESP32C3_WIFI_CONNECT_TIMEOUT
+	int "Connect timeout by second"
+	default 10
+	help
+		Max waiting time of connecting to AP.
+
+config ESP32C3_WIFI_SAVE_PARAM
+	bool "Save WiFi Parameters"
```

Comment:
I think should include a "depends on !DISABLE_MOUNTPOINT"

Reply:
@donghengqaz please fill all the field [here](https://github.com/apache/incubator-nuttx/pull/3040#issue-591426880)

Reply:
Yes, I will add this.